airflow/www/decorators.py

```diff
@@ -44,36 +44,36 @@ def _mask_variable_fields(extra_fields):
     Mask the 'val_content' field if 'key_content' is in the mask list.
 
     The variable requests values and args comes in this form:
-    [('key', 'key_content'),('val', 'val_content'), ('description', 'description_content')]
+    [{'key': 'key_content'},{'val': 'val_content'}, {'description': 'description_content'}]
```

Comment:
```suggestion
    {'key': 'key_content', 'val': 'val_content', 'description': 'description_content'}
```

Isn't this a single dict?

Reply:
Yes, good catch. I forgot to update the comment after I changed it from an array to a single dict